File: 문자열/백준_1475_방번호/방번호.cpp
#include <iostream>

using namespace std;

int main() {
    int n = 0;
    scanf("%d", &n);
    int arr[10] ={0};
    while(n >=1){
        int i = n %10;
        if(i == 9) i = 6;
        arr[i]++;
        n/=10;
    }
    arr[6] = (arr[6] % 2 == 0) ? arr[6]/2 : arr[6]/2+1;
    int max = 0;
    for(int i = 0; i< 10; i++) if(arr[i] > max) max = arr[i];
    printf("%d", max);

}

File: 문자열/백준_1475_방번호/방번호.py
n = int(input())
arr = [0 for _ in range(0, 10)]

while n >= 1:
    i = n % 10
    if i == 9: i = 6
    arr[i] += 1
    n //= 10

arr[6] = arr[6] // 2 if arr[6] % 2 == 0 else arr[6] // 2 + 1
_max = 0
for i in arr:
    if i > _max: _max = i
print(_max)

